wideq/core.py

```diff
@@ -240,166 +345,218 @@ def parse_oauth_callback(url):
     """
 
     params = parse_qs(urlparse(url).query)
-    return params['access_token'][0], params['refresh_token'][0]
+    return params["oauth2_backend_url"][0], params["code"][0], params["user_number"][0]
 
 
-def login(api_root, access_token, country, language):
-    """Use an access token to log into the API and obtain a session and
-    return information about the session.
+def auth_request(oauth_url, data):
+    """Use an auth code to log into the v2 API and obtain an access token 
+    and refresh token.
     """
+    auth_path = "/oauth/1.0/oauth2/token"
+    url = urljoin(oauth_url, "/oauth/1.0/oauth2/token")
```

Comment:
Any particular reason this path is written out twice instead of using `auth_path` twice? (Also maybe this should be a constant.)